stack/values.yaml

```diff
@@ -118,3 +118,8 @@ kube-prometheus-stack:
         enabled: false
 opentelemetry-operator:
   enabled: true
+  kubeRBACProxy:
+    enabled: true
+    image:
+      repository: gcr.io/kubebuilder/kube-rbac-proxy
+      tag: v0.13.0
```

Comment:
Few questions:
- Why do we need it?
- Why not enable it by defult in tobs?
- Can we not specify image tag here? I fear it will go stale fairly quickly.

Reply:
Eh well not sure on this one.  Let me try again without it, but I was getting download errors on the version (`v0.11.0`).  Let me run through the scenario again without this and see what happens.  Could have been my wifi or something.